src/components/Providers/ProviderNavigation/ProviderNavigation.js

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+import ArrowRight from '../../../assets/icons/arrow-right.png';
+import { formatProviderHeading } from '../../../helpers/utils';
+import './ProviderNavigation.scss';
+
+const ProviderNavigation = ({ name, title }) => {
+  const providerHeading = formatProviderHeading(name, title);
+
+  return (
+    <nav className="profile__breadcrumb">
+      <Link to="/providers" className="providers__link">
+        Mental Wellness
```

Comment:
How could we extend the app to be configurable to search for different types of providers? Ie. Physiotherapists.